src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -1933,11 +1932,12 @@ export class StudyViewPageStore {
     }
 
     readonly unfilteredClinicalDataCount = remoteData<ClinicalDataCountItem[]>({
+        await:() => [this.selectedSampleIdentifiersFilter],
         invoke: () => {
             return internalClient.fetchClinicalDataCountsUsingPOST({
                 clinicalDataCountFilter: {
                     attributes: this.unfilteredAttrsForNonNumerical,
-                    studyViewFilter: this.filters,
+                    studyViewFilter: this.selectedSampleIdentifiersFilter.result!,
```

Comment:
are these not already contained in this.filters?   it seems strange that we maintain some filter state outside of the filters.

Reply:
`selectedSampleIdentifiersFilter` is a `studyViewFilter` with just `SampleIdentifiers` based on the `this.filters`.  sending `this.filters` for every api request would calculate filtered sample every time. instead if we pass just the `SampleIdentifiers` then this would save lot of computations and improve response time. Updated pr's description

Reply:
I see.   Would it make any sense to rename selectedSamples to "filteredSamples" ?